src/visudialog.cpp

```diff
@@ -0,0 +1,112 @@
+﻿#include "visudialog.h"
+
+VisuDialog::VisuDialog()
+{
+	setWindowTitle("Data needed");
```

Comment:
Not very explicit.
Maybe something like "Visualization parameters" would be more explicit?